eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1871,20 +1871,25 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
         case _ => ()
       }
 
+      val previousChannelUpdate = stateData match {
```

Comment:
The `_opt` suffix is a convention we have for options.

```suggestion
      val previousChannelUpdate_opt = stateData match {
```